test/e2e/helpers/assertions.ts

```diff
@@ -0,0 +1,24 @@
+import { browser } from '@wdio/globals'
+import type { Workbench } from 'wdio-vscode-service'
+
+/**
+ * Asserts that the CLI Commands and Patches sections are visible in the sidebar. This is
+ * considered the default state when the workspace is open with a git and rad initialized
+ * repository.
+ */
+export async function expectCliCommandsAndPatchesToBeVisible(workbench: Workbench) {
```

Comment:
I was thinking of having a more semantic name for this function. Something that expresses that it is the default view of the extension in the sidebar (e.g. `expectDefaultSidebarViewToBeVisible`).
That way, if sections are added, removed in the future, this name remains relevant. Thoughts?

Reply:
Good point. While both work, I think the alternative more semantic name would better match how we use the func and scales better over time.

Though it may be more accurate to use a name like `expectStandardSidebarViewsToBeVisible`.

Reply:
Renamed as suggested